### PR TITLE
NCI-Agency/anet#3698: Fix buttons

### PR DESCRIPTION
**Pull request by gjvoosten**

### CODE REVIEW

File: client/src/components/AdvancedSearch.js [hunk 1]
```diff
@@ -216,18 +212,15 @@ const AdvancedSearch = ({
                 }}
               >
                 <Button
-                  className={Classes.POPOVER_DISMISS}
-                  intent="danger"
```
Comment (asafkaganbezgin) [inline client/src/components/AdvancedSearch.js:220]: We can use `outline-danger` if some buttons **should** indicate a dangerous event.

File: client/src/components/EditHistory.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { Button } from "@blueprintjs/core"
+import { Button, Intent } from "@blueprintjs/core"
```
Comment (asafkaganbezgin) [inline client/src/components/EditHistory.js:1]: I think we should completely use Bootstrap buttons to be consistent in UI. Using two different button types from two different UI libraries does not make sense to me.

Comment (asafkaganbezgin) [reply client/src/components/EditHistory.js:1]: Sorry for not bringing that up before you start fixing buttons.

File: client/src/components/AdvancedSearch.js (hunk 1 above)
Comment (gjvoosten) [reply client/src/components/AdvancedSearch.js:220]: This was mixing BlueprintJS props with React Bootstrap buttons (probably copy-paste remnants). The props were meaningsless in this case.

File: client/src/components/EditHistory.js (hunk 2 above)
Comment (gjvoosten) [reply client/src/components/EditHistory.js:1]: I was halfway replacing those and then thought it better to leave this as a separate action. So put that on the TODO list.

**Pull request merged**